src/Raven.Client/Documents/Operations/TimeSeries/TimeSeriesCollectionConfiguration.cs

```diff
@@ -22,20 +22,31 @@ public class TimeSeriesCollectionConfiguration : IDynamicJson
         /// </summary>
         public RawTimeSeriesPolicy RawPolicy { get; set; } = RawTimeSeriesPolicy.Default;
 
-        internal void Initialize()
+        internal void ValidateAndInitialize()
         {
             if (Policies.Count == 0)
                 return;
 
             Policies.Sort(TimeSeriesDownSamplePolicyComparer.Instance);
```

Comment:
This is now wrong, no?
`TimeSeriesDownSamplePolicyComparer` isn't taking into account the `Months`

Reply:
We use the upper and lower bound of number of seconds in a month, so we compare it if we can, otherwise an exception is thrown.